extraction/data_models/embedded_instagram_post.py

```diff
@@ -0,0 +1,33 @@
+"""This class creates the object structure for Embedded Instagram content"""
+
+from data_models.contents import ContentType, _Content
+
+
+class EInstagramPost(_Content):
+    """This class creates Embedded Instagram object"""
+
+    def __init__(self, code):
+        super(EInstagramPost, self).\
+            __init__(ContentType.EMBEDDED_INSTAGRAM_POST)
+        self.insta_shortcode = code
+        self.media_type = ''
+
+    def __str__(self):
+        content_string = super(EInstagramPost, self).__str__()
+        return '{}{{EInstagram:\n\t\t\t' \
+               '{{shortcode: {shortcode}\n\t\t\t' \
+               'media_type: {media_type}}}\n\t\t}}'.\
+            format(content_string, shortcode=self.insta_shortcode,
+                   media_type=self.media_type)
+
+    def set_media_type(self, media_type):
+        """This method sets the type of media like Image, Video, etc"""
+        self.media_type = media_type
+
+    def get_shortcode(self):
```

Comment:
there's no setter for this, is that intended? 

Also in the classes below, some properties have setters and getters, while some have only getters -  how are we deciding which should have which?

Reply:
Also, if we're using getters and setters, we should probably hide the actual variable (by prepending with _).

https://docs.python.org/3/library/functions.html#property
@property can also be used to create readonly variables.

If the setter/getters are not doing anything special - we can just remove them.

Reply:
Removed getters and setters!